epitator/infection_annotator.py

```diff
@@ -262,9 +260,70 @@ def from_noun_chunks_with_infection_lemmas(doc, debug=False):
     return(infection_spans)
 
 
+def collapse_span_group(span_group):
+    if len(span_group.base_spans) == 0:
+        # Not a span group
+        return span_group
+    collapsed_metadata = {
+        'attributes': []
+    }
+    if span_group.label:
+        collapsed_metadata['attributes'] = [span_group.label]
+    for span in span_group.base_spans:
+        span = collapse_span_group(span)
+        if span.metadata:
+            collapsed_metadata = dict(
+                dict(collapsed_metadata, **span.metadata),
+                attributes=collapsed_metadata['attributes'] + span.metadata['attributes'])
+        # if span.label:
+        #     collapsed_metadata['attributes'].append(span.label)
+    collapsed_metadata['attributes'] = sorted(collapsed_metadata['attributes'])
+    return AnnoSpan(span_group.start, span_group.end, span_group.doc, metadata=collapsed_metadata)
+
+
+def add_count_modifiers(spans, doc):
+    spacy_tokens, spacy_nes = doc.require_tiers('spacy.tokens', 'spacy.nes', via=SpacyAnnotator)
+    span_tier = AnnoTier(spans)
+    spacy_lemmas = [span.token.lemma_ for span in spacy_tokens]
+
+    def search_lemmas(lemmas, match_name=None):
+        match_spans = []
+        lemmas = set(lemmas)
+        for span, lemma in zip(spacy_tokens, spacy_lemmas):
+            if lemma in lemmas:
+                match_spans.append(span)
+        return AnnoTier(match_spans, presorted=True).label_spans(match_name)
+
+    case_statuses = (
+        search_lemmas(['suspect'], 'suspected') +
+        search_lemmas(['confirm'], 'confirmed'))
+    span_tier += span_tier.with_nearby_spans_from(case_statuses, max_dist=1)
+
+    person_and_place_nes = spacy_nes.with_label('GPE') + spacy_nes.with_label('PERSON')
+    modifier_lemma_groups = [
```

Comment:
I've seen you use this pattern elsewhere — I think it's nifty, using the first lemma as the label. It's more succinct than the dict method I used above.